app/views/api_docs/pages/release_notes.md

```diff
@@ -1,3 +1,13 @@
+### 7th July 2020
+
+Documentation has been amended to emphasise the stability of `/applications` endpoints
+in contrast to the experimental and unsupported nature of `/test-data` endpoints.
```

Comment:
"experimental and unsupported" sounds like we _really_ don't care! Like, we do support them. Maybe just "in contrast to" is enough?

Reply:
@duncanjbrown I've revised the apocalyptic language and added a 308 redirect for the `/experimental` routes.